manifests/d/DuckStudio/GitHubView/Nightly/2025.01.22.12907017964/DuckStudio.GitHubView.Nightly.installer.yaml

```
@@ -0,0 +1,39 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: DuckStudio.GitHubView.Nightly
+PackageVersion: 2025.01.22.12907017964
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- ghv
+Protocols:
+- https
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.Runtime.8
+ProductCode: GitHubView_is1
+ReleaseDate: 2025-01-22
+AppsAndFeaturesEntries:
+- ProductCode: GitHubView_is1
+ElevationRequirement: elevatesSelf
```

```suggestion
ReleaseDate: 2025-01-22
```

我在 iss 文件中没有指定 ProductCode。
当程序以用户范围安装时，不需要提升权限。

---

抱歉，我没有发现这个安装程序和我之前几个程序的安装程序的“独特之处”
- https://github.com/microsoft/winget-pkgs/pull/215274#discussion_r1935976599